hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOffloadableSetUnlockOperation.java

```diff
@@ -104,7 +108,8 @@ public boolean returnsResponse() {
 
     @Override
     public Operation getBackupOperation() {
-        return entryBackupProcessor != null ? new EntryBackupOperation(name, dataKey, entryBackupProcessor) : null;
+        return entryBackupProcessor != null
```

Comment:
Do we not need to apply the TTL to the backup as well?

Reply:
no it is handled by entry-backup-processor.